backend/controllers/userController.js

```diff
@@ -0,0 +1,61 @@
+const asyncHandler = require("express-async-handler");
+const generateToken = require("../utils/jsonUtils");
+const User = require("../models/userModel");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken")
+
+const registerUser = asyncHandler(async (req, res) => {
+  console.log("HERE");
+  const { name, email, password } = req.body;
```

Comment:
Similar to the note made in the mongo schema file, not sure if we'll be needing the name value since the frontend currently won't be supplying this value

Reply:
I think the front end should be supplying the name attribute while registering a user. But I will remove the requirement of the name attribute nevertheless.

Reply:
Fixed: Changes are reflected in recent commits